fecgov/fec-eregs
# Add new menus + about menu, implement glossary on eregs

Resolves: #407, #409
- Implemented glossary on Eregs, matching fec-cms styles as closely as possible given the differing styles of eregs
- Added new mega-menu designs and added About menu which was not there.
### [How to test &#187;](#how-to-test)

There are still some responsive issues with the mega menu that are present on production as well:
- [ ]  All menu items default to open below the medium breakpoint (currently like this on production)
- [ ]  There are some minor spacing issues where the menu and the masthead content meet (currently like this on production)
 For a followup issue I am going to take a pass at solving this by removing the handlebars templates for the menus and replacing them with normal partials( like we do on fec-cms). I seem to remember @xtine saying that we no longer needed to use handlebars for the menus on production and perhaps it was just never also changed on Eregs.
<img width="1661" alt="screen shot 2018-07-06 at 3 16 36 pm" src="https://user-images.githubusercontent.com/5572856/42396453-aeea20ce-812f-11e8-845f-f1d7ab4bdb79.png">

<a id="how-to-test"></a> 

### How to test:|
- Checkout this branch locally `feature/update-headers-fix-glossary-link`
- `npm install`
- `npm run build`
- `./python manage.py compile_frontend`
- `./manage.py runserver`
- go to http://127.0.0.1:8000/
- Check the mega menu and make sure the about menu is there
- Test the glossary
Thanks for your time!
